packages/typescript/ai/tests/strip-to-spec-middleware.test.ts

```
@@ -0,0 +1,244 @@
+import { describe, it, expect } from 'vitest'
```

ESLint

[error] Member 'expect' of the import declaration should be sorted alphabetically.

(sort-imports)